pkg/kubefed2/unjoin.go

```diff
@@ -167,10 +173,15 @@ func UnjoinCluster(hostConfig, clusterConfig *rest.Config, federationNamespace,
 		return err
 	}
 
-	clusterClientset, err := util.ClusterClientset(clusterConfig)
-	if err != nil {
-		glog.V(2).Infof("Failed to get unjoining cluster clientset: %v", err)
-		return err
+	var clusterClientset *client.Clientset
+	if clusterConfig != nil {
+		clusterClientset, err = util.ClusterClientset(clusterConfig, forceDeletion)
+		if err != nil {
```

Comment:
This error handling is redundant and can be removed, as per a previous comment about `util.ClusterClientset`.